lib/ansible/utils/module_docs_fragments/spacewalk.py

```diff
@@ -0,0 +1,31 @@
+# Copyright: (c) 2018, Davide Blasi (@davegararh)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+    # Parameters for Sapcewalk modules
+    DOCUMENTATION = '''
+options:
```

Comment:
Do you want to mention the other environment variables, like you have for `C(SPACEWALK_VALIDATE_CERTS)`

Reply:
Added